src/renderer.test.ts

```diff
@@ -126,6 +126,16 @@ describe('renderer returns expected content', () => {
 
         expect(text.flat().length).toBe(7);
     });
+
+    test ('Renders caption node types', () => {
+        const text = renderCaption(JSDOM.fragment('this caption contains <em>html</em>, <br>html, <strong>html</strong>'),  Pillar.news);
```

Comment:
```suggest
const text = renderCaption(JSDOM.fragment('this caption contains'),  Pillar.news);
```

We're only testing the first item so I guess we don't need to have the other types?

Reply:
Ah I realise this will likely reduce the test coverage though

Reply:
👍